src/blocks/Message.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import CommonMessage, { CommonMessageProps } from "./shared/CommonMessage"
+
+import "./Message.scss"
+
+interface MessageProps extends Omit<CommonMessageProps, "role"> {}
+
+const Message = (props: MessageProps) => {
+  const classNames = ["message"]
+  if (props.className) classNames.push(props.className)
+
+  return <CommonMessage {...props} role="region" className={classNames.join(" ")} />
```

Comment:
I don't think the region role makes sense here (looking at these docs https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role `Use sparingly[! Landmark roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#landmark_roles) are intended to be used sparingly, to identify larger overall sections of the document.` It also looks like when a region role is needed, a label is needed as well.

Reply:
Good point. Should we take that off here as a default but consumers can provide those props if they need to?

Reply:
I can't see a way where these components would be used in a way where they would need a role, I personally think we can remove it!